downstream/modules/platform/ref-single-eda-controller-with-internal-db.adoc

```diff
@@ -27,17 +23,9 @@ automationedacontroller_pg_database='automationedacontroller'
 automationedacontroller_pg_username='automationedacontroller'
 automationedacontroller_pg_password='<password>'
 
-# The full routable URL used by EDA to connect to a controller host.
```

Comment:
Hi @jamesmarshall24 @jself-sudoku I may missing context. Why we remove these comments? it looks like a valid info for the customer.

Reply:
@Alex-Izquierdo I don't really know do we want to keep them in or not? It seems like most of the examples on this page leave out comments from the inventory file while others keep them.

Reply:
@jself-sudoku I would say this is your call. I do think maybe we should keep them in to give context to the variables. I'll give a follow up suggestion change to what Alejandro asked.

Reply:
@jamesmarshall24 and @Alex-Izquierdo - I only removed those to comply with the example that @jamesmarshall24 added to the comments in the jira. I can add them back with no problem. It's just a matter of undoing what was completed before. As for me making the call, I don't feel comfortable doing that at this point. I'm hoping to gain more knowledge as we progress, but I'm definitely not there yet. So, I'll pass this through to you both after I've added those lines back.